lib/new_version.dart

```diff
@@ -107,15 +115,15 @@ class NewVersion {
     } else if (Platform.isAndroid) {
       return _getAndroidStoreVersion(packageInfo);
     } else {
-      debugPrint(
-          'The target platform "${Platform.operatingSystem}" is not yet supported by this package.');
+      debugPrint('The target platform "${Platform.operatingSystem}" is not yet supported by this package.');
     }
   }
 
-  /// This function attempts to clean local version strings so they match the MAJOR.MINOR.PATCH
+  /// This function attempts to clean local version strings so they match the MAJOR.MINOR.PATCH+BUILD
   /// versioning pattern, so they can be properly compared with the store version.
-  String _getCleanVersion(String version) =>
-      RegExp(r'\d+\.\d+\.\d+').stringMatch(version) ?? '0.0.0';
+  String _getCleanVersion(String version) {
+    return RegExp(r'\d+\.\d+\.\d+\+\d+').stringMatch(version) ?? '0.0.0+0';
```

Comment:
```suggestion
    return RegExp(r'\d+\.\d+\.\d+(\+\d+)?').stringMatch(version) ?? '0.0.0+0';
```

Regex to allow the build info to be optional and thus match the simpler notation as well.